svnwiki-mode.el

```diff
@@ -1,11 +1,13 @@
-;;; svnwiki-mode.el --- Major mode for editing svnwiki markup
+;;; svnwiki-mode.el --- Major mode for editing svnwiki markup -*- lexical-binding: t -*-
 
 ;; Copyright (C) 2016 Vasilij Schneidermann <mail@vasilij.de>
+;; SPDX-License-Identifier: GPL-3.0-or-later
 
 ;; Author: Vasilij Schneidermann <mail@vasilij.de>
-;; URL: https://depp.brause.cc/svnwiki-mode
+;; URL: https://github.com/chicken-contrib/svnwiki-mode
 ;; Version: 0.0.1
-;; Keywords: text
+;; Package-Requires: ((emacs "24.1"))
```

Comment:
Are you sure that's the correct minimum version? This package makes use of rx and multiline fontification.

Reply:
I'm not sure. 24 is the first version with lexical-binding, and package-lint doesn't warn about it. There's a [docker container](https://github.com/Silex/docker-emacs) for 24.5 so we could check whether it works with that version.

Reply:
I've verified, 24.1 has rx and multiline fontification. According to the NEWS that is.